_rules/error-field-is-not-identified-using-color-only-i3z4v8.md

```diff
@@ -0,0 +1,166 @@
+---
+id: i3z4v8
+name: Error field is not identified using color differences only
+rule_type: atomic
+description: |
+  This rule checks that error fields can be identified not only by color differences but through another visual means.
+accessibility_requirements:
+  wcag21:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  -  DOM Tree
+  -  CSS Styling
+acknowledgments:
+  authors:
+    -  Giacomo Petri
+---
+
+## Description
+
+This rule checks that errors are not identified only through color differences.
+
+## Applicability
+
+This rule applies to any [visible][] [input error][] together with:
+- its [programmatic label]; and
+- the closest error message that refers to it.
```

Comment:
I'm also not sure that "error message" is unambiguous.
For example, putting a red cross next to errored field and a green check mark next to good ones, is a red cross an "error message" (it's not really a message)? (or the red border of Passed Example 3)